src/index.ts

```diff
@@ -60,11 +76,25 @@ export class Aracari<T extends HTMLElement = HTMLElement> {
     return this;
   }
 
-  public remap() {
-    this.mapping = this.getTextNodeMapping(this.root);
+  public remap(mapping?: Mapping) {
+    this.mapping = mapping ?? this.getTextNodeMapping(this.root);
     return this;
   }
 
+  private getNodeByAddress (address: string) {
+    const path = address.split(".").map(i => parseInt(i, 10));
+    return this.walkNodes(this.root, path);
+  }
+
+  private getMappingForText (text: string, caseSenative: boolean = true): string[] | undefined {
+    const pattern = new RegExp(text, `${caseSenative ? 'i' : ''}g`);
```

Comment:
I think caseSensitive may be a sp error?

Reply:
haha yeah been there but I can update.

Reply:
updated